frontend/packages/data-portal/app/components/Filters/SelectFilter.tsx

```diff
@@ -33,15 +34,19 @@ export function SelectFilter<
 }: {
   className?: string
   details?: ReactNode
-  groupBy?: (option: Value<Option, Multiple>) => string
+  groupBy?: (
+    option: AutocompleteValue<Option, Multiple, DisableClearable, FreeSolo>,
```

Comment:
is it possible for these to be typed as `SdsOption`?

Reply:
CC: @masoudmanson for help here!

Reply:
@tihuan , we could definitely export some type aliases, but I believe encapsulating those generic parameters with fixed values directly in SDS might not be the best approach.

@bchu1 , does it make sense to define a type alias in your specific context and use it within the component? This way, it remains flexible and context-specific.

```js
// Single Select
type SDSOption = AutocompleteValue<Option, false, false, false>;

// Or maybe a Multi Select
type SDSOptionMultiple = AutocompleteValue<Option, true, false, false>;
```

Reply:
oh yeah, I just meant can we use our existing type alias

Reply:
Thanks so much for the feedback, @codemonkey800 💡 !

Yeah I don't remember there was a strong reason for SDS to make `name` a required prop in the option and deviate from MUI's default `label` as the required prop. In general it feels like following MUI's default makes sense and when SDS deviates some MUI, it needs to have a strong justification

Actually @sidneymbell ran into the same issue a year ago or so too. So since that's now a recurring pattern, maybe it's worth the breaking change?

@masoudmanson what do you think?

Reply:
@tihuan Absolutely! I’ll add this to the biweekly agenda so we can discuss the breaking change with the rest of the team and find out if there was a specific reason for making the "name" prop required in the option. 😬🥳

Reply:
Thanks so much, @masoudmanson ! Now thinking about it, I might have been the one who introduced "name" as required years ago, thinking that `label` is not as intuitive as name 🤦 💀 So if no one else objects, we should just go with `label` 😆

Reply:
Haha, no worries at all 😯😬

I’ll bring this up in the breaking changes channel to make sure everyone’s aligned on switching from `name` to `label` and `id`. If there are no objections, I’ll get started on a PR to make the update.

Reply:
Thank you so much!! I regret changing it from label to name years ago now 😭 😆